lenisha/scribe
# [TELUSHAS-59][TELUSHAS-73] - Template Dropdown, Redesign

### 📝 Description

- Completely redesign the app
- Moved each box (Transcription, Clinical Note, Patient Handout) into their own tabs, and separated the controls/options
- Refactored the useDictation hook into a context since it now needs to be shared
- Added the template dropdown

### 🔗 Link to JIRA Ticket

[Template Dropdown](https://willowtree.atlassian.net/browse/TELUSHAS-59)
[Redesign](https://willowtree.atlassian.net/browse/TELUSHAS-73)

### 🧪 Testing Notes

- Make sure the app works as before with the new layout
- Verify the selected template is being passed in the Clinical Note 'AI Generate' backend call (being passed in the body)
- 
### 📸 Screenshots / 🎥 Video

- New layout

https://github.com/user-attachments/assets/d0ebbee6-8ee3-46fe-9899-24f795abcda2


- Template dropdown

![Screenshot 2024-12-20 at 3 17 13 PM](https://github.com/user-attachments/assets/e211acf3-ef23-4cce-91b9-2f1ad30e1f84)
